controller/src/main/java/org/dswarm/controller/resources/BasicResource.java

```diff
@@ -583,4 +553,61 @@ protected Response createUpdateResponse(final PROXYPOJOCLASS proxyObject, final
 
 		return responseBuilder.entity(objectJSON).build();
 	}
+
+	private Response createObjectInternal(final PROXYPOJOCLASS proxyObject) throws DMPControllerException {
+
+		if (proxyObject == null) {
+
+			BasicResource.LOG.debug("couldn't add {}", pojoClassName);
+			throw new DMPControllerException("couldn't add " + pojoClassName);
+		}
+
+		final POJOCLASS object = proxyObject.getObject();
+
+		if (object == null) {
+
+			BasicResource.LOG.debug("couldn't add {}", pojoClassName);
+			throw new DMPControllerException("couldn't add " + pojoClassName);
+		}
+
+		BasicResource.LOG.debug("added new {} with id = '{}' ", pojoClassName, object.getUuid());
+		BasicResource.LOG.trace(" = '{}'", ToStringBuilder.reflectionToString(object));
```

Comment:
`ToStringBuilder.reflectionToString` is very expensive, can you wrap this in an enabled-check?